todo_app/data/task.py

```diff
@@ -0,0 +1,10 @@
+class Task:
+    
+
+    def __init__(self, id, title, description = "", status = "To Do", duedate="") -> None:
```

Comment:
We don't really need `None` here
```suggestion
    def __init__(self, id, title, description = "", status = "To Do", duedate=""):
```